packages/framework-integration-tests/package.json

```diff
@@ -48,7 +49,7 @@
     "watch:local": "nodemon --watch ../framework-provider-local/dist --watch ../framework-provider-local-infrastructure --watch dist --exec \"../cli/bin/run start -e local\"",
     "lint:check": "eslint --ext '.js,.ts' **/*.ts",
     "lint:fix": "eslint --quiet --fix --ext '.js,.ts' **/*.ts",
-    "compile": "tsc -b tsconfig.json",
+    "compile": "npx ttsc -b tsconfig.json",
```

Comment:
I think this is the right command. Please, check the other pacakge.json because I think you used `ttypescript` instead of `ttsc`

Reply:
Idk why but it was failing in the other one when using `ttsc`, that's why it's changed.

Reply:
Check the other comment, I think it could be related to installing dependencies when generating the user project.